### ISSUE TITLE
can risc0 support deep learning model inference ZKP?

### ISSUE THREAD

**Issue by chaoyanghe** [risc0/risc0#279]

**Comment by brianretford** [comment]
Yes, it can, though we haven't actually done any of it. One could imagine lots of paths to achieving this, including compiling and running ONNX. I think we're a major release or two away from compellingly being able to support AI use cases though - currently computations are limited to a generally insufficient number of cycles (that will be addressed in future  releases though)
